src/hb-ot-name.h

```diff
@@ -48,6 +48,16 @@ typedef unsigned int hb_name_id_t;
  **/
 #define HB_NAME_ID_INVALID 0xFFFF
 
+HB_EXTERN unsigned int
+hb_ot_name_get_string (hb_face_t     *face,
+		       unsigned int   name_id,
+		       hb_language_t *languages,
+		       unsigned int   languages_count,
+		       unsigned int   start_offset,
+		       unsigned int  *buffer_length /* IN/OUT.  May be NULL */,
+		       char          *buffer        /* OUT.     May be NULL */);
```

Comment:
Too complicated.  Need one language.  Also, get_utf8 and get_utf16 variants.  See my API details in other issue.  Sketch it there first, not in a patch.  Thanks.  Also, hb_name_id_t.

Reply:
Also, don't think we need start_offset, since encoding has to walk through the whole thing...